### DISCUSSION TITLE
Opendrift in operational deployments

### DISCUSSION THREAD

**Discussion by John-Luick** [OpenDrift/opendrift#533] [General]
This paper: "Massively parallel implicit equal-weights particle filter for ocean drift trajectory forecasting" by Holm et al seems to imply that opendrift has been set up to run operationally (offline) with a Met Norway ocean model. Is it possible to view this in operation online, and has the process been documented or published?

**Comment by knutfrode** [comment]
That was a research-project, so operational there just meant "auomated and experimental".

But MET Norway have an "official" web service based on OpenDrift, in beta: https://beta.drifty.met.no
Note that use of this is password protected, for registered users.
There is presently no more documentation of the service than what is available on those pages.

**Comment by John-Luick** [comment]
Thanks. It looks like very promising. Mind you, I did not know an "API" from the "CIA" until reading a bit of the OpenAPI documentation. I will leave it at that as I am unlikely to be needing simulation results from Met Norway models, but overall it was a bit of an eye-opener. I will just mention that it might be useful for Matlab users to interact directly with the api. (Also maybe useful if they populated the "Examples" section!)

**Comment by gauteh** [comment]
I converted this issue to a discussion since running opendrift in operational settings might be of general interest. At MET there is one setup, but there are others, so that maybe some general recommendations and experiences can be given.

**Reply by gauteh** [reply]
Or maybe this is covered by https://github.com/OpenDrift/opendrift/discussions/469 already?
